src/docker-osx-dev

```diff
@@ -1031,6 +1033,23 @@ function install {
 }
 
 #
+# Fetch the latest docker-osx-dev script, and then run install
+#
+function update_self {
+  log_info "Updating to the latest docker-osx-dev"
+  current_file=$(which docker-osx-dev)
+  log_info "  -- Backing up $current_file to $current_file.old"
```

Comment:
Probably a good idea to put `"$current_file.old"` into a variable to keep things more DRY